src/elm/Main.elm

```diff
@@ -174,6 +183,27 @@ update msg model =
         ScrollResult _ ->
             ( model, Cmd.none )
 
+        Pile1 msg_ ->
+            let
+                ( pile1, cmd ) =
+                    Pile.update Pile1 msg_ model.pile1
+            in
+            ( { model | pile1 = pile1 }, cmd )
+
+        Pile2 msg_ ->
+            let
+                ( pile2, cmd ) =
+                    Pile.update Pile2 msg_ model.pile2
+            in
+            ( { model | pile2 = pile2 }, cmd )
+
+        Pile3 msg_ ->
```

Comment:
These commands all follow the same pattern, is there an easy way to make it one command and use section id to designate which Pile?